test/Engine/ProtoTest/ProtoTest.csproj

```diff
@@ -211,6 +211,9 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="Test.ds" />
```

Comment:
Should this be `testclass.ds`, but then why do you need to include it in the project?

Reply:
Not needed. Will remove this